packages/dashboard/src/customization/propertiesSections/gaugeStyleSettings/displaySettingsSection.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { Box, Checkbox, FormField } from '@cloudscape-design/components';
+
+import { PropertyLens } from '~/customization/propertiesSection';
+import { GaugeProperties } from '~/customization/widgets/types';
+import { DashboardWidget } from '~/types';
+import { StyledExpandableSection } from '../components/styledComponents';
+import { maybeWithDefault } from '~/util/maybe';
+
+export const DisplaySettingsSection = ({
```

Comment:
instead of writing a new `DisplaySettings` section specific to the Gauge component could we update the existing one 
which is found at `dashboard/src/customization/propertiesSections/displaySettingsSection/index.tsx`